proto/iot.proto

```diff
@@ -144,6 +165,32 @@ message ThermostatScheduleBlock {
     string time = 4;
 }
 
+message ThermostatADREvent {
+    //End of History Date Range
+    //unit: seconds
+    Int64 event_end = 1;
+    //Start of History Date Range
+    //unit: seconds
+    Int64 event_start = 2;
+    //Current event type of DR Event
+    //unit: xbos/iot/DREventType
+    DREventType event_type = 3;
+    //Current system status of DR event
+    //unit: xbos/iot/DREventStatus
+    DREventStatus dr_status = 4;
+    //Time when event takes effect
+    //unit: seconds
+    Int64 time = 5;
+}
+
+message ThermostatOccupancy {
```

Comment:
What does it mean for a thermostat to be occupied? What's the intended use case of this message? We may just want to stick with a generic occupancy sensor class

Reply:
This is based on the occupancy.go file of the Pelican driver (Linked [here](https://github.com/john-b-yang/bw2-contrib/blob/master/driver/pelican/types/occupancy.go)). That code is just for indicating whether a driver is "occupied" or not, so I agree that this message might be overcompensating. Maybe a better alternative is just to add an "occupancy" field to the Thermostat message?

Reply:
I believe the occupancy portion of the Pelican driver has to do with extracting that information out of the Pelican API, but not necessarily the thermostats. I think the right way to go here is to have the Pelican driver publish a generic occupancy sensor message, so you can just remove this message.

Reply:
Got it, makes sense. I'll keep this in mind when updating the Pelican driver too. I think your reason makes sense just because the information conveyed by occupancy is pretty small, so it's not really necessary. Removed this message!